python/GafferImageUI/RGBAChannelsPlugValueWidget.py

```diff
@@ -47,7 +47,7 @@ class RGBAChannelsPlugValueWidget( GafferUI.PlugValueWidget ) :
 	def __init__( self, plug, **kw ) :
 
 		self.__menuButton = GafferUI.MenuButton( menu = GafferUI.Menu( Gaffer.WeakMethod( self.__menuDefinition ) ) )
-		self.__menuButton._qtWidget().setMinimumWidth( 150 )
+		self.__menuButton._qtWidget().setFixedWidth( 175 )
```

Comment:
@johnhaddon I don't _think_ `setTextElideMode()` is supported on buttons, unless I'm missing something. This felt like a reasonable compromise in width. Its used in two other UIs : `ImageSamplerUI` and `ImageStatsUI`.

I couldn't find a use of `ImageSamplerUI` unless my `grep` foo is failing me, and it does make the NodeEditor for `ImageStats` look a bit rubbish now:

![image](https://user-images.githubusercontent.com/896779/79779592-85865c00-8332-11ea-94ed-84f7529dca02.png)

Will try to find an alternative approach.

Reply:
Fixed properly in https://github.com/GafferHQ/gaffer/pull/3723/commits/a1068bc4b8edf2e35c73f76e8421121dcfff643d